pathd/path_pcep_cli.c

```diff
@@ -43,6 +43,8 @@
 #define DEFAULT_TIMER_SESSION_TIMEOUT_INTERVAL 30
 #define DEFAULT_DELEGATION_TIMEOUT_INTERVAL 10
 
+#define BUFFER_PCE_SIZE 1024
```

Comment:
IMO the naming is misleading, PCE, but PCC buffer also :shrug:

Reply:
lets call it BUFFER_PCC_PCE_SIZE. this deals with comm between, PCC and PCE.